src/legends.js

```diff
@@ -40,7 +40,7 @@ export function exposeLegends(scales, context, defaults = {}) {
 }
 
 function legendOptions({className, ...context}, {label, ticks, tickFormat} = {}, options) {
-  return inherit(options, {className: `${className}-legend`, ...context}, {label, ticks, tickFormat});
+  return inherit(options, {className, ...context}, {label, ticks, tickFormat});
```

Comment:
This suffixing is moved down to the legend implementation because we need a distinct suffix for each legend implementation to avoid conflicting styles.